pdns/pdnsutil.cc

```diff
@@ -4407,96 +4407,255 @@ static int backendLookup(vector<string>& cmds)
+  {"list-algorithms", {false, listAlgorithms, GROUP_DNSSEC,
+   "list-algorithms [with-backend]     List all DNSSEC algorithms supported, optionally also listing the crypto library used"}},
+  {"list-all-zones", {true, listAllZones, GROUP_ZONE,
+   "list-all-zones [primary|secondary|native|producer|consumer]\n"
+   "                                   List all active zone names. --verbose or -v will also include disabled or empty zones"}},
+  {"list-autoprimaries", {true, listAutoprimaries, GROUP_AUTOPRIMARY,
+   "list-autoprimaries                 List all autoprimaries"}},
+  {"list-keys", {true, listKeys, GROUP_DNSSEC,
+   "list-keys [ZONE]                   List DNSSEC keys for ZONE. When ZONE is unset, display all keys for all active zones"}},
+  {"list-member-zones", {true, listMemberZones, GROUP_ZONE,
+   "list-member-zones CATALOG          List all members of catalog zone CATALOG"}},
+  {"list-tsig-keys", {true, listTSIGKeys, GROUP_TSIGKEY,
+   "list-tsig-keys                     List all TSIG keys"}},
+  {"list-zone", {true, listZone, GROUP_ZONE,
+   "list-zone ZONE                     List zone contents"}},
+  {"lmdb-get-backend-version", {false, lmdbGetBackendVersion, GROUP_OTHER,
+   "lmdb-get-backend-version           Get schema version supported by backend"}},
+  {"load-zone", {true, loadZone, GROUP_ZONE,
+   "load-zone ZONE FILE                Load ZONE from FILE, possibly creating zone or atomically\n"
+   "                                   replacing contents\n"
+   "                                   --verbose or -v will also include the keys for disabled or empty zones"}},
+  {"publish-zone-key", {true, publishZoneKey, GROUP_ZONEKEY,
+   "publish-zone-key ZONE KEY-ID       Publish the zone key with key id KEY-ID in ZONE"}},
+  {"raw-lua-from-content", {true, rawLuaFromContent, GROUP_OTHER,
+   "raw-lua-from-content TYPE CONTENT  Display record contents in a form suitable for dnsdist's `SpoofRawAction`"}},
+  {"rectify-all-zones", {true, rectifyAllZones, GROUP_DNSSEC,
+   "rectify-all-zones [quiet]          Rectify all zones. Optionally quiet output with errors only"}},
+  {"rectify-zone", {true, rectifyZone, GROUP_DNSSEC,
+   "rectify-zone ZONE [ZONE ..]        Fix up DNSSEC fields (order, auth)"}},
+  {"remove-autoprimary", {true, removeAutoprimary, GROUP_AUTOPRIMARY,
+   "remove-autoprimary IP NAMESERVER   Remove an autoprimary"}},
+  {"remove-zone-key", {true, removeZoneKey, GROUP_ZONEKEY,
+   "remove-zone-key ZONE KEY-ID        Remove key with KEY-ID from ZONE"}},
+  {"replace-rrset", {true, replaceRRSet, GROUP_RRSET,
+   "replace-rrset ZONE NAME TYPE [ttl] Replace named RRSET from zone\n"
+   "       content [content..]"}},
+  {"secure-all-zones", {true, secureAllZones, GROUP_DNSSEC,
+   "secure-all-zones [increase-serial] Secure all zones without keys"}},
+  {"secure-zone", {true, secureZone, GROUP_DNSSEC,
+   "secure-zone ZONE [ZONE ..]         Add DNSSEC to zone ZONE"}},
```

Comment:
```suggestion
   "secure-zone ZONE [ZONE ...]        Add DNSSEC to zone ZONE"}},
```

And what happens if there's more than one zone?

Reply:
They all get processed.